rdkit/Chem/Descriptors.py

```diff
@@ -88,6 +88,39 @@ def _setupDescriptors(namespace):
   31.05...
 
 """
+def GetNumChiralCenters(mol):
+  """The number of chiral centers in a molecule
+  >>>> GetNumChiralCenters(Chem.MolFromSmiles('CC(N)O'))
+  1
+  >>>> GetNumChiralCenters(Chem.MolFromSmiles('CC(Br)C(N)O'))
+  2
+  
+  """
+  chiralcenters = Chem.FindMolChiralCenters(mol, includeUnassigned=True)
+  return len(chiralcenters)
```

Comment:
There is not really a need to copy the result of the function to a new array and determine the length of that. It could be that Python is optimizing this out, I don't know.